korangar/src/main.rs

```diff
@@ -1144,7 +1144,7 @@ impl Client {
                 NetworkEvent::UpdateEquippedPosition { index, equipped_position } => {
                     self.player_inventory.update_equipped_position(index, equipped_position);
                 }
-                NetworkEvent::ChangeJob(account_id, job_id) => {
+                NetworkEvent::ChangeSprite(account_id, sprite_type, value, value2) => {
```

Comment:
This is not something I documented anywhere but `NetworkEvent`s should be a bit more abstract than the ragnrok online packets. I would like to keep the fact that changing the job and changing the hair sprite is sent via the same packet as an implemenation detail.

So in this case, I would like to keep the old `ChangeJob` event and _add_ a second event called `ChangeHead` (or something along these lines) to keep them separated.